nmostesting/NMOSTesting.py

```diff
@@ -340,6 +341,20 @@
     #     }],
     #     "class": IS1001Test.IS1001Test
     # },
+    "IS-12-01": {
+        "name": "IS-12 NMOS Control Protocol",
+        "specs": [{
+            "spec_key": "is-04",
+            "api_key": "node",
+            "disable_fields": ["selector"]
+        }, {
+            "spec_key": "is-12",
+            "api_key": "ncp",
+            "websocket": True,

```

Comment:
Probably this is OK for first iteration,  but what do you think about whether `websocket` and `selector` flags belong in the spec definitions rather than the test suite definitions?